hs_core/views/utils.py

```diff
@@ -1272,7 +1238,19 @@ def unzip_file(user, res_id, zip_with_rel_path, bool_remove_original,
         if unzip_to_folder:
```

Comment:
The `unzipped_folder_paths` here (https://github.com/hydroshare/hydroshare/pull/5578/files#diff-c6a5e98cf78fb7f12dc0bf2255241aed8abaeb7e48b0c563a3cc5acdfa581fe8R1228) will not include any empty folders in the original zip file.

Reply:
hmm, nice catch. Do you believe that is worth implementing? It seems like an unlikely edge scenario.